ltr/concourse/deploy.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+import os
+import sagemaker
+
+from datetime import date
+
+govuk_environment = os.environ["GOVUK_ENVIRONMENT"]
+
+s3_bucket = os.getenv("S3_BUCKET", f"govuk-{govuk_environment}-search-relevancy")
+endpoint_name = os.getenv(
+    "ENDPOINT_NAME", f"govuk-{govuk_environment}-search-ltr-endpoint"
+)
+
+role = sagemaker.get_execution_role()
+session = sagemaker.Session()
+
+# find the current endpoint config
+current_endpoint_config_name = session.boto_session.client(
+    "sagemaker"
+).describe_endpoint(endpoint_name)["EndpointConfigName"]
+current_endpoint_config = session.boto_session.client(
+    "sagemaker"
+).describe_endpoint_config(EndpointConfigName=current_endpoint_config_name,)
+
+# deploy the model by creating a new endpoint config and updating the existing endpoint
+model = sagemaker.tensorflow.serving.Model(
+    f"s3://{s3_bucket}/model/{date.today().isoformat()}",
```

Comment:
Actually, I wonder if we want to think about the version names now, since it's probably one of those things that is handy to get right first time. What do you think?

Reply:
We could include the time in the model name, and also include the name of the training data.  We could even check that all the training data is of the same version.

Reply:
I've changed the training data name to be the current UNIX timestamp, and the model name to be `{training name}-{UNIX timestamp}`